components/clm/src/data_types/VegetationDataType.F90

```diff
@@ -8070,12 +8070,12 @@ subroutine veg_cf_summary(this, bounds, num_soilp, filter_soilp, num_soilc, filt
             this%psnshade_to_cpool(p)
 
        ! maintenance respiration (MR)
-       if ( trim(isotope) == 'c13' .or. trim(isotope) == 'c14') then
+       !if ( trim(isotope) == 'c13' .or. trim(isotope) == 'c14') then

```

Comment:
Please delete the line instead of leaving as a comment and reindent the following lines. thanks

Reply:
Done as you suggested. Thanks